app/manage/pods/device-details/share-modal-directive/index.js

```diff
@@ -11,10 +11,12 @@ module.exports = [
       controller: [
         '$scope',
         '$timeout',
-        function($scope, $timeout) {
+        '$location',
+        function($scope, $timeout, $location) {
+
           // template variables -------------------
           $scope.copySuccess = false;
-          $scope.url = window.location.hostname + window.location.pathname;
+          $scope.url = $location.absUrl().split('?')[0];
```

Comment:
perhaps, yeah. the `?` is in the wrong part of the URL anyways for the `demo=1`

I think maybe leave this as is and enable whatever it is that allows URLs to not have the `#` in it. I know that was a thing that came up before